### DISCUSSION TITLE
Phpstorm & Webstorm usage Tailwind

### DISCUSSION THREAD

**Discussion by alikdb** [tailwindlabs/tailwindcss#6183] [Help]
Hello, I am using Jetbrains app. This is how it looks to me.
![autocomplate](https://user-images.githubusercontent.com/44875945/143010900-f89cd059-dabf-417a-bbcb-a0d2879d0635.png)

but i want like below image
![autocomplate](https://user-images.githubusercontent.com/44875945/143011474-ca11dfa0-e0ee-4f66-b100-32cc519e8b66.png)

**Comment by shmshd** [comment]
[Intelijj Tailwind CSS plugin](https://plugins.jetbrains.com/plugin/15321-tailwind-css) is built by Jetbrains. You are comparing VS Code extension with the Intelijj plugin built by two different org/people. I don't think anybody here can help with that. Maybe you can ask this question on their [issue board](https://youtrack.jetbrains.com/issues) as a feature request?
